apps/src/applab/applab.js

```diff
@@ -699,8 +703,8 @@ Applab.onMouseMoveDebugResizeBar = function (event) {
   //  we explicitly set bottom/height styles on the elements above, we need to
   //  do the same for these styles as well)
 
-  codeTextbox.style.minHeight = 'calc(50% - 21px)';
-  debugArea.style.maxHeight = 'calc(50% - 21px)';
+  codeTextbox.style.minHeight = 'calc(50% - 16px)';
```

Comment:
It is weird, but the comment above reads:

```javascript
  // (we would do this in CSS, but for precedence rules to work properly, if
  //  we explicitly set bottom/height styles on the elements above, we need to
  //  do the same for these styles as well)
```

Perhaps these properties _should_ be in CSS, marked `!important`?